src/main/java/Main.java

```diff
@@ -1,7 +1,116 @@
-import java.io.IOException;
+import clear_all_use_case.*;
+import complete_task_use_case.*;
+import create_event_use_case.*;
+import create_task_use_case.*;
+import delete_event_use_case.*;
+import delete_task_use_case.*;
+import entities.EventFactory;
+import entities.TaskFactory;
+import modify_event_use_case.*;
+import modify_task_use_case.*;
+import screens.*;
+
+import javax.swing.*;
+import java.util.HashMap;
+import java.util.Map;
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Test");
+
+        // Initialize create event classes
+        CreateEventOutputBoundary createEventPresenter = new CreateEventPresenter();
+        CreateEventDsGateway createEventDsGateway = new CreateEventDataAccess();
+        EventFactory eventFactory = new EventFactory();
+        CreateEventInputBoundary createEventInteractor = new CreateEventInteractor(
+                createEventDsGateway, createEventPresenter, eventFactory);
+        CreateEventController createEventController = new CreateEventController(createEventInteractor);
+
+        // Initialize modify event classes
+        ModifyEventOutputBoundary modifyEventOutputBoundary = new ModifyEventPresenter();
+        ModifyEventDsGateway modifyEventDsGateway = new ModifyEventDataAccess();
+        ModifyEventInputBoundary modifyEventInputBoundary = new ModifyEventInteractor(modifyEventOutputBoundary,
+                modifyEventDsGateway);
+        ModifyEventController modifyEventController = new ModifyEventController(modifyEventInputBoundary);
+
+        // Initialize delete event classes
+        DeleteEventOutputBoundary deleteEventOutputBoundary = new DeleteEventPresenter();
+        DeleteEventDsGateway deleteEventDsGateway = new DeleteEventDataAccess();
+        DeleteEventInputBoundary deleteEventInputBoundary = new DeleteEventInteractor(deleteEventOutputBoundary,
+                deleteEventDsGateway);
+        DeleteEventController deleteEventController = new DeleteEventController(deleteEventInputBoundary);
+
+        // Initialize create task classes
+        CreateTaskOutputBoundary createTaskOutputBoundary = new CreateTaskPresenter();
+        CreateTaskDsGateway createTaskDsGateway = new CreateTaskDataAccess();
+        TaskFactory taskFactory = new TaskFactory();
+        CreateTaskInputBoundary createTaskInputBoundary = new CreateTaskInteractor(taskFactory,
+                createTaskOutputBoundary, createTaskDsGateway);
+        CreateTaskController createTaskController = new CreateTaskController(createTaskInputBoundary);
+
+        // Initialize modify task classes
+        ModifyTaskOutputBoundary modifyTaskPresenter = new ModifyTaskPresenter();
+        ModifyTaskDataAccess dsGateway = new ModifyTaskDataAccess();
```

Comment:
This should be a ModifyTaskDsGateway instance, although it is initialized with an object of type ModifyTaskDataAccess.

Reply:
Thanks for catching that. Fixed it.